test_data/rinkeby-example.csv

```diff
@@ -8,4 +8,5 @@ token_address,receiver,amount
 0x4dbcdf9b62e891a7cec5a2568c3f4faf9e8abe2b,0x7000000000000000000000000000000000000000,0.0007
 0x4dbcdf9b62e891a7cec5a2568c3f4faf9e8abe2b,0x8000000000000000000000000000000000000000,0.0008
 0x784b46a4331f5c7c495f296ae700652265ab2fc6,0x9000000000000000000000000000000000000000,0.01
-0x784b46a4331f5c7c495f296ae700652265ab2fc6,0xa000000000000000000000000000000000000000,0.02
\ No newline at end of file
+0x784b46a4331f5c7c495f296ae700652265ab2fc6,0xa000000000000000000000000000000000000000,0.02
+0x4dcf5ac4509888714dd43a5ccc46d7ab389d9c23,0xa000000000000000000000000000000000000000,2
```

Comment:
Whats this fancy new token?

Reply:
```suggestion
0x4dcf5ac4509888714dd43a5ccc46d7ab389d9c23,0xb000000000000000000000000000000000000000,2
```

Reply:
THE HUMAN TOKEN HMT haha It was the first on rinkeby.etherscan.io I could find.